src/runtime.jl

```diff
@@ -1,5 +1,7 @@
 using ChainRulesCore
 
+struct DiffractorRuleConfig <: RuleConfig{Union{HasReverseMode,HasForwardsMode}} end
```

Comment:
This doesn't seem a great fit for the terse style of this package, better ideas are welcome.

Reply:
You mean the name? Seems fine to me.

Reply:
Yes, sorry somehow I lost the word.